duckduckgo/iOS
# Simplify the tracker icon asset loading code

Task/Issue URL: https://app.asana.com/0/414709148257752/1204033732341887/f

**Description**:
Background
The code for loading matching logo or first letter icon for blocked trackers animation was heavily based on original macOS implementation. It was build using hugely repetitive switch statement. Additionally the code contained typo and was not properly loading a dark mode variant for one of the logos. Since iOS implementation uses automatic support for loading dark mode assets we can simplify the code.

**Steps to test this PR**:
- Test if logo icons load for known trackers by their entity name.
- Ensure that icons for trackers containing space in their name ("the trade desk" and "verizon media") are loaded properly
- Check if trackers without matching logo load generic first letter icon instead

**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
